packages/sdk/src/HopBridge.ts

```diff
@@ -2013,7 +2012,7 @@ class HopBridge extends Base {
       sourceChain
     )
 
-    const feeBps = this.getFeeBps(this.tokenSymbol, destinationChain)
+    const feeBps = await this.getFeeBps(this.tokenSymbol, destinationChain)
```

Comment:
I do not believe this needs to be `await`ed. I think the `async` keyword can be removed from `getFeeBps`.

Reply:
Resolved by https://github.com/hop-protocol/hop/pull/326/commits/79f100a3e97a6a2fb07c9b2dcab5a6ad405d85de